activerecord/lib/active_record/railtie.rb

```diff
@@ -29,10 +29,22 @@ class Railtie < Rails::Railtie # :nodoc:
       'ActiveRecord::RecordNotSaved'   => :unprocessable_entity
     )
 
-
     config.active_record.use_schema_cache_dump = true
     config.active_record.maintain_test_schema = true
 
+    def (config.active_record).destroy_all_in_batches=(value)
+      if value == false
```

Comment:
Issue the warning when the value is used, so it'll point to their `destroy_all` call